### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by V-Soni**
Added new links and done spellcheck

### CODE REVIEW

**Comment by unnown** [comment]
Please mark as invalid and spam, this is an obvious attempt to get a free t-shirt by 'contributing'(hacktoberfest)
This will also ban the user for trying to abuse the system by doing marginal effort

**Comment by V-Soni** [comment]
> Please mark as invalid and spam, this is an obvious attempt to get a free t-shirt by 'contributing'(hacktoberfest)
> This will also ban the user for trying to abuse the system by doing marginal effort

Please go and read the changes made before commenting

**Comment by unnown** [comment]
I did, hence the words MARGINAL, not allowed by rules of hacktober, 
Please go and read those rules before you attempt to spam

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This repo contains sample use case projects using the [OpenTok Insights API].
 
 ## Welcome to OpenTok
 
-If you're new to OpenTok, you can [sign up for a Tokbox account](https://tokbox.com/account/user/signup?utm_source=DEV_REL&utm_medium=github&utm_campaign=) and get some free credit to get you started.
```
Comment (lukeocodes) [inline README.md:7]: This wasn't a typo as credit can be valid both singular and plural

**Review by lukeocodes — CHANGES_REQUESTED** [review]
Hi. While I appreciate the contribution, I feel its a little light on content. If you cannot provide more, I'll mark it as `invalid`. I disagree this is `spam` though.

**Comment by lukeocodes** [comment]
@unnown perhaps your comments will be received better if your profile was not both anonymous and empty. We're capable of handling PRs too, thank you.

File: README.md (hunk 1 above)
Comment (V-Soni) [reply README.md:7]: Okay Sure Sir
I will try to contribute more

**Pull request closed without merging**